### PR TITLE
Add Azure Pipeline and Remove Jenkins builds

### PR DESCRIPTION
**Pull request by papowerNI**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-data-sharing-framework-custom-device/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

* Remove the build line from the Jenkinsfile
* Add azure-pipeline.yaml to point to main archive location
* Remove build.toml
* Add a post-build step to Core projects to move the PPL outputs to the correct directory (previously this was done by Jenkins)

### Why should this Pull Request be merged?

Jenkins will not be used soon.

### What testing has been done?

PR Build outputs should be in the right places.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        - version: '2023'
+          bitness: '64bit'
+
+      buildSteps:
+        - projectLocation: 'Source\Core\Data Sharing Framework Core Build - Windows.lvproj'
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'My Computer'
+          buildSpec: 'Data Sharing Framework Core'
+
+        - projectLocation: 'Source\Core\Data Sharing Framework Core Build - Linux RT.lvproj'
```
Comment (Karl-G1) [inline azure-pipelines.yml:33]: This needs moved to right before the Linux engine build of the Custom Device (next to last build).

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'My Computer'
+          buildSpec: 'Scripting Examples'
+
+        - projectLocation: 'Source\Custom Device\Data Sharing Framework Custom Device.lvproj'
+          buildOperation: 'ExecuteBuildSpec'
+          target: 'My Computer'
+          buildSpec: 'Engine Release'
+
+        - projectLocation: 'Source\Core\Data Sharing Framework Core Build - Linux RT.lvproj'
```
Comment (Karl-G1) [inline azure-pipelines.yml:58]: Needs to point to the custom device project: `Source\Custom Device\Data Sharing Framework Custom Device Linux RT x64.lvproj`

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+          payloadMaps:
+            - installLocation: 'documents\National Instruments\NI VeriStand $(lvVersion)\Custom Devices\Data Sharing Framework'
+              payloadLocation: 'Source\Built\Custom Device'
+            - installLocation: 'ni-paths-NISHAREDDIR$(nipkgx64suffix)\Errors\English'
+              payloadLocation: 'Source\Built\Errors'
+
+        - controlFileName: 'control_dsf_scripting'
+          payloadMaps:
+            - installLocation: 'ni-paths-LV$(lvVersion)DIR$(nipkgx64suffix)\vi.lib\addons\VeriStand Custom Device Scripting APIs\Data Sharing Framework'
+              payloadLocation: 'Source\Built\Custom Device'
```
Comment (Karl-G1) [inline azure-pipelines.yml:78]: Needs to point to the Scripting output directory: 'Source\Built\Scripting API'

**Review by Karl-G1 — CHANGES_REQUESTED** [review]

**Comment by Karl-G1** [comment]
- [ ] build.toml is not actually deleted it appears
- [ ] Let's go ahead and bump the version to 23.5

File: azure-pipelines.yml (hunk 1 above)
Comment (papowerNI) [reply azure-pipelines.yml]: 01e9e359ff20174dcff22eaafbeeacf29e12c076

File: azure-pipelines.yml (hunk 2 above)
Comment (papowerNI) [reply azure-pipelines.yml:55]: 01e9e359ff20174dcff22eaafbeeacf29e12c076

**Review by Karl-G1 — APPROVED** [review]

**Pull request merged**